docker/post_github_comment.py

```diff
@@ -61,7 +103,7 @@ def main():
     inspect_format = f'''
 {{{{ range $env := .Config.Env }}}}{{{{ if eq $env "DEPRECATED_IMAGE=true" }}}}## 🔴 IMPORTANT: This image is deprecated 🔴{{{{ end }}}}{{{{ end }}}}
 ## Docker Metadata
-- Image Size: `{get_docker_image_size(args.docker_image)}`
+- Image Size: `{get_docker_image_size(args.docker_image, is_contribution=args.is_contribution)}`
```

Comment:
`is_contribution` arg was added to support getting the image size from the CircleCI artifacts when running in a contribution mode. If not, we get the error seen [here](https://app.circleci.com/pipelines/github/demisto/dockerfiles/83325/workflows/ee5becc4-861e-41b6-9e04-490cd39fff6a/jobs/89478/steps?invite=true#step-106-336):

```
[1] failed getting image size for image: devdemisto/bottle2:1.0.0.89478. Err: 404 Client Error: Not Found for url: https://hub.docker.com/v2/repositories/devdemisto/bottle2/tags/1.0.0.89478/
```

Reply:
The change result can be seen [here](https://app.circleci.com/pipelines/github/demisto/dockerfiles/83365/workflows/f30e83bf-0a4c-4024-a74c-0cfb490b8cc1/jobs/89518?invite=true#step-106-353) or https://github.com/demisto/dockerfiles/pull/23484#issuecomment-1981384890.